src/CardContent.js

```diff
@@ -5,20 +5,40 @@ function CardContent() {
   // Pull in the date and then break it into parts to be passed into
   // the card and then rendered into the App
 
-  const [data, setData] = useState([]);
+  const [copyData, setCopyData] = useState([]);
```

Comment:
turning all of this into 1 array of objects would have been better (which you probably know now)